src/txn/txn_ckpt.c

```diff
@@ -287,8 +287,9 @@ __wt_checkpoint_get_handles(WT_SESSION_IMPL *session, const char *cfg[])
              * rollback error. We will ignore this dhandle as part of this checkpoint by returning
              * from here.
              */
-            WT_IGNORE_RET(__wt_msg(
-              session, "WT_ROLLBACK: checkpoint raced with transaction operating on dhandle"));
+            reason = "checkpoint raced with transaction operating on dhandle";
+            __wt_verbose_debug(session, WT_VERB_TRANSACTION, "%s", reason);
```

Comment:
I'm a little bit confused why we log the same message via `__wt_verbose_debug` & `__wt_msg`. If the debug level is enabled for `WT_VERB_TRANSACTION`, doesn't that mean we print the same message twice?
i.e. would it be better to just have `__wt_msg` for now (until we at least implemented a `wt_verbose_x` replacement for  `__wt_msg`).

Reply:
I believe your comment holds for `src/cursor/cur_file.c` as well. I will remove the `__wt_verbose_debug` calls. This was effectively what I was wondering, hence our conversation last week about `__wt_msg`.

Reply:
All good! Sorry for the confusion, thanks for the fix :)